src/DETHRACE/common/utility.c

```diff
@@ -443,6 +443,30 @@ br_pixelmap* PurifiedPixelmap(br_pixelmap* pSrc) {
     NOT_IMPLEMENTED();
 }
 
+#if defined(DETHRACE_FIX_BUGS)
+static void align_pixelmap(br_pixelmap* pm) {
+    int i;
+    br_uint_16 original_row_bytes;
+    br_uint_16 new_row_bytes;
+    br_uint_8* original_pixels;
+    br_uint_8* new_pixels;
+
+    original_row_bytes = pm->row_bytes;
+    if (original_row_bytes % sizeof(int) == 0 || pm->height <= 1) {
+        return;
+    }
+    new_row_bytes = (original_row_bytes + sizeof(int) - 1) & ~(sizeof(int) - 1);
```

Comment:
i wonder if everywhere we are using `sizeof(int) - 1) & ~(sizeof(int) - 1)` we should be using `int32_t` instead of `int`. We always expect that value to be "4" regardless of the size of an int

Reply:
Yes good point.

Reply:
Added this in b9d600b